templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -14,9 +14,16 @@ contents:
     counter=0
     # find default interface
     while [ $counter -lt 12 ]; do
+      # check ipv4
       iface=$(ip -j route show default | jq -r '.[0].dev')
-      if [ -n "$iface" ]; then
-        echo "Default gateway interface found: ${iface}"
+      if [[ -n "$iface"  && "$iface" != "null" ]]; then
```

Comment:
fwiw you can do `jq -r '.[0].dev // empty'` to avoid the "`null`" output
(you can also use `jq -e` to exit with non-0 status on "`null`", but I don't think that would help here)